src/components/pages/PointShare/RenderPointShare.js

```diff
@@ -1,84 +1,64 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useOktaAuth } from '@okta/okta-react/dist/OktaContext';
-import { Header } from '../../common';
-import { Row, Col, InputNumber, Button, notification } from 'antd';
+import { Button } from 'antd';
 import { QuestionCircleOutlined } from '@ant-design/icons';
 import { connect } from 'react-redux';
 import { submitPoints } from '../../../api/index';
 
-import { SubmissionViewerModal } from '../../common';
-import { InstructionsModal } from '../../common';
+import { Header, SubmissionViewerModal, InstructionsModal } from '../../common';
 import { modalInstructions } from '../../../utils/helpers';
 
+import ChildRow from './ChildRow';
+import usePointShare from './usePointShare';
+
 const PointShare = props => {
   const { push } = useHistory();
-  const [totalPoints, setTotalPoints] = useState(100);
-  const [storyOnePoints, setStoryOnePoints] = useState(0);
-  const [storyTwoPoints, setStoryTwoPoints] = useState(0);
-  const [illustrationOnePoints, setIllustrationOnePoints] = useState(0);
-  const [illustrationTwoPoints, setIllustrationTwoPoints] = useState(0);
-  const [teamPoints, setTeamPoints] = useState(null);
+
+  const [pointsLeft, portfolioPoints, handleUpdatePoints] = usePointShare();
+  // TODO could probably abstract this into a useModal hook
   const [modalContent, setModalContent] = useState(null);
   const [showModal, setShowModal] = useState(false);
   const [modalVisible, setModalVisible] = useState(true);
 
+  // TODO The previous component used a useEffect that would send the submit API call if authState changed, I'm unsure if it's needed -- I haven't re-implemented it yet
```

Comment:
Seems like it should be implemented.

Reply:
Ok, I can re-implement it then.  The way I was thinking was like: 
If authstate changes to be authorized then I wasn't sure why it would call the submit handler, and if authstate changes to be unauthorized then I wasn't sure if it would just call the submit handler and return an unauthorized error.  Am I misinterpreting it? @drew-ross

Reply:
Spoke over zoom. All good